pypowsybl/network.py

```diff
@@ -186,6 +186,12 @@ def __init__(self, handle: _pp.JavaHandle):
         self._source_format = att.source_format
         self._forecast_distance = _datetime.timedelta(minutes=att.forecast_distance)
         self._case_date = _datetime.datetime.utcfromtimestamp(att.case_date)
+        self.elements = {"Bus":self.get_buses().shape[0],"Busbarsections":self.get_busbar_sections().shape[0],
+                         "Gens" : self.get_generators().shape[0],"Load":self.get_loads().shape[0],
+                         "Lines":self.get_lines().shape[0],
+                         "Trafo2w":self.get_2_windings_transformers().shape[0],
```

Comment:
We need to be as much as possible consistent in the naming of things accross the lib:
- 2 windings transformers instead of trafo2w
- same for 3 windings
- generators instead of "gens"

Reply:
yes i will correct this